packages/dataviews/src/components/dataviews-view-config/index.tsx

```diff
@@ -251,32 +253,42 @@ function ItemsPerPageControl() {
 	);
 }
 
-function FieldItem( {
-	field,
+function BaseFieldItem( {
+	fieldId,
+	label,
+	subLabel,
 	isVisible,
 	isFirst,
 	isLast,
 	canMove = true,
+	canHide = true,
+	isInteracting = false,
 	onToggleVisibility,
 	onMoveUp,
 	onMoveDown,
+	additionalActions,
 }: {
-	field: NormalizedField< any >;
```

Comment:
Why remove this and add more individual props? If anything we should aim for the opposite if possible IMO.

Reply:
Hi @ntsekouras, because the "Preview" item we create with a label sublabel/description etc is not a field, this component receives strings that can come from real field properties.